[EDU-3118] feat: add secure infrastructure (#464)
aziontech/docs: src/content/docs/en/pages/secure-journey/secure-infrastructure.mdx

---
title: Secure an infrastructure
description: Use Origin Shield to secure your infrastructure.
meta_tags: edge, security, infrastructure, origin shield
namespace: docs_secure_infrastructure
menu_namespace: secureMenu
permalink: /documentation/products/secure/secure-infrastructure/
---

import Button from '~/components/Button.astro'

Once you've [created an edge firewall and secured your application](/en/documentation/products/secure/secure-application/), you can use **Origin Shield** to secure your infrastructure by blocking threats.

> Origin Shield works as a list of IP/CIDR addresses used in Azion's Edge, thus providing ensurance of safe IPs to allow access. The list is maintained by Azion and available via Real-Time Manager (RTM) and API, but each client must create automations to guarantee their security policies are always up-to-date with the Origin Shield addresses.

Reviewer: ```suggestion
Origin Shield works as a list of IP/CIDR addresses used in Azion's edge, thus providing assurance of safe IPs to allow access. The list is maintained by Azion and available via Real-Time Manager (RTM) and API, but each client must create automation to guarantee their security policies are always up-to-date with the Origin Shield addresses.
```

---

## Step 1. Enable Origin Shield

To access the Origin Shield list, you must have:

- An [edge firewall created](/en/documentation/products/secure/secure-application/).
- The [Network Layer Protection module](/en/documentation/products/guides/secure/configure-main-settings/#modules) enabled.

---

> ## Step 2. Access the list of addressess

Reviewer 2: ```suggestion
## Step 2. Access the list of addresses
```

2. Select **Azion IP Tor Exit Nodes**.
3. On the **List** field, you can copy the IP addresses list.

:::note
Clients with Origin Shield receive an email every time the list is updated.
:::

### Via API

1. Run the following `GET` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/)

```bash
curl --location 'https://api.azionapi.net/network_lists' \
--header 'Accept: application/json; version=3' \
--header 'Authorization: Token [TOKEN VALUE]'
```

2. You'll receive a response similar to this:

```json
{
  "count": 4,
  "total_pages": 1,
  "schema_version": 3,
  "links": {
    "previous": null,
    "next": null
  },
  "results": [
    {
      "id": 2345,
      "last_editor": "last.editor@azion.com",
      "last_modified": "2023-03-27T21:19:27.996870Z",
      "list_type": "ip_cidr",
      "name": "Azion IP Tor Exit Nodes",
      "country_list": [],
      "ip_list": [
        "192.168.0.5"
      ]
    }
  ]
}
```

3. On the `ip_list` field, you'll receive all IPs added by Azion. You can copy the IP addresses list.

---

> ## Step 3. Automate new addresses additions

Reviewer 2: ```suggestion
## Step 3. Automate new address additions
```
